source/position/position.go

```diff
@@ -34,43 +35,51 @@ type Position struct {
 	// IteratorType - shows in what iterator was created position.
 	IteratorType IteratorType
 
+	// Snapshot information.
+	// SnapshotLastProcessedVal - last processed value from ordering column.
+	SnapshotLastProcessedVal any
+	// SnapshotMaxValue max value by  ordering column, when snapshot starts work.
+	SnapshotMaxValue any
```

Comment:
So, essentially this is what was needed to fix the bug in #22?

Reply:
Yes  and also it resolve issue with acceptance test